pkg/ssh/server/ssh.go

```diff
@@ -173,6 +173,12 @@ func (s *Server) handler(sess ssh.Session) {
 	ptyReq, winCh, isPty := sess.Pty()
 	cmd := s.getCommand(sess, isPty)
 	if ssh.AgentRequested(sess) {
+		// on some systems (like containers) /tmp may not exists, this ensures
+		// that we have a compliant directory structure
+		err := os.MkdirAll("/tmp", os.ModePerm)
```

Comment:
Please use 0o777 here as os.ModePerm can cause issues on other operating systems like macos (I know this shouldn't happen but we should try to avoid it)

Reply:
done